### PR TITLE
Chore: update pot file

### PR DESCRIPTION
**Pull request by bradp**
## Description
Makes some minor fixes in translated strings as well as updating the pot file.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php [hunk 1]
```diff
@@ -712,7 +712,7 @@ public function stripe_connect_field( $value, $option_value ) {
 					if ( give_stripe_is_connected() ) :
 						$site_url            = get_site_url();
 						$stripe_user_id      = give_get_option( 'give_stripe_user_id' );
-						$modal_title         = __( '<strong>You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.</strong>', 'give' );
+						$modal_title         = '<strong>' . __( 'You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.', 'give' ) . '</strong>';
```
Comment (ravinderk) [inline includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php:715]: @bradp can we use `esc_html__`

Comment (bradp) [reply includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php:715]: It doesn't make sense to me to pass in HTML for the translators, as we don't want the markup changed.

Comment (ravinderk) [reply includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php:715]: @bradp I agree on this but can we replace `__` with `esc_html__`

File: includes/misc-functions.php [hunk 2]
```diff
@@ -594,25 +594,25 @@ function array_column( $input = null, $columnKey = null, $indexKey = null ) {
...
 		if ( ! is_int( $params[1] ) && ! is_float( $params[1] ) && ! is_string( $params[1] ) && $params[1] !== null && ! ( is_object( $params[1] ) && method_exists( $params[1], '__toString' ) ) ) {
-			trigger_error( esc_html__( 'array_column(): The column key should be either a string or an integer.', 'give' ), E_USER_WARNING );
+			trigger_error( 'array_column(): The column key should be either a string or an integer.',, E_USER_WARNING );
 
 			return false;
 		}
 
 		if ( isset( $params[2] ) && ! is_int( $params[2] ) && ! is_float( $params[2] ) && ! is_string( $params[2] ) && ! ( is_object( $params[2] ) && method_exists( $params[2], '__toString' ) ) ) {
-			trigger_error( esc_html__( 'array_column(): The index key should be either a string or an integer.', 'give' ), E_USER_WARNING );
+			trigger_error( 'array_column(): The index key should be either a string or an integer.',, E_USER_WARNING );
```
Comment (DevinWalker) [inline includes/misc-functions.php:615]: @bradp I think you have an extra comma here.

Comment (bradp) [reply includes/misc-functions.php]: Fixed!

**Comment by bradp** [comment]
Issues fixed, should be good now @ravinderk @DevinWalker

File: includes/misc-functions.php [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?php
+reCAPTCHA<?php
```
Comment (ravinderk) [inline includes/misc-functions.php:1]: @bradp typo?

Comment (bradp) [reply includes/misc-functions.php]: Oops

**Pull request merged**